tcms/issuetracker/azure_integration.py

```diff
@@ -0,0 +1,79 @@
+# -*- coding: utf-8 -*-
+from requests.auth import HTTPBasicAuth
+import requests
+
+from tcms.issuetracker import base
+
+
+class AzureAPI():
+    def __init__(self, base_url=None, password=None):
+        self.api_version = "?api-version=6.0"
+        self.headers = {"Accept": 'application/json-patch+json',
+                        'Content-type': 'application/json-patch+json'}
+        self.auth = HTTPBasicAuth('apikey', password)
+        self.base_url = base_url + "/_apis/"
+
+    def get_issue(self, issue_id):
+        url = "{0}{1}{2}{3}".format(self.base_url, "wit/workitems/",
+                                    str(issue_id), self.api_version)
+        response_json = self._get(url, self.headers, self.auth)
+        return response_json
+
+    def create_issue(self, body):
+        url = "{0}{1}{2}".format(self.base_url, "wit/workitems/$Issue", self.api_version)
+        response_json = self._post(url, self.headers, self.auth, body)
+        return response_json
+
+    def update_issue(self, issue_id, body):
+        url = "{0}{1}{2}{3}".format(self.base_url, "wit/workitems/",
+                                    str(issue_id), self.api_version)
+        response_json = self._patch(url, self.headers, self.auth, body)
+        return response_json
+
+    def add_comment(self, issue_id, body):
+        url = "{0}{1}{2}{3}{4}{5}".format(self.base_url, "wit/workitems/",
+                                          str(issue_id), "/comments", self.api_version,
+                                          "-preview.3")
+        response_json = self._post(url, self.headers, self.auth, body)
+        return response_json
+
+    def get_comments(self, issue_id):
+        url = "{0}{1}{2}{3}{4}{5}".format(self.base_url, "wit/workitems/",
+                                          str(issue_id), "/comments", self.api_version,
+                                          "-preview.3")
+        response_json = self._get(url, self.headers, self.auth)
+        return response_json
+
+    @staticmethod
+    def _get(url, headers, auth):
+        response = requests.get(url, headers=headers, auth=auth)
+        response_json = response.json()
+        return response_json
+
+    @staticmethod
+    def _post(url, headers, auth, body):
+        response = requests.post(url, headers=headers, auth=auth, json=body)
+        response_json = response.json()
+        return response_json
+
+    @staticmethod
+    def _patch(url, headers, auth, body):
+        response = requests.patch(url, headers=headers, auth=auth, json=body)
+        response_json = response.json()
```

Comment:
IMO the intermediate `response_json` variable doesn't bring anything useful so you can user return directly.

Reply:
This comment applies to all methods of the AzureAPI class.